<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>influenceVertexAreaQWidget</class>
 <widget class="QWidget" name="influenceVertexAreaQWidget">
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="influenceVertexAreaQLabel">
     <property name="text">
      <string>Vertex Influence Area:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="influenceVertexAreaQList">
     <property name="selectionMode">
      <enum>QAbstractItemView::NoSelection</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="influenceVertexAreaSelectQButton">
     <property name="text">
      <string>Select Content</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="influenceVertexAreaAddQButton">
       <property name="text">
        <string>Add Selection</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="influenceVertexAreaSetQButton">
       <property name="text">
        <string>Set Selection</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
